applications/settings/power_settings_app/scenes/power_settings_scene_power_off.c

```diff
@@ -9,11 +10,20 @@ void power_settings_scene_power_off_dialog_callback(DialogExResult result, void*
 void power_settings_scene_power_off_on_enter(void* context) {
     PowerSettingsApp* app = context;
     DialogEx* dialog = app->dialog;
+    DolphinSettings settings = dolphin_get_settings();
```

Comment:
exposing DolphinSettings internals is not good. I'd stick with getters and setters